core/primitives/src/delegate_action.rs

```diff
@@ -74,7 +75,8 @@ impl DelegateAction {
     }
 
     pub fn get_hash(&self) -> CryptoHash {
```

Comment:
The naming of this function feels a bit confusing

Reply:
Good point. I renamed it to `get_nep461_hash` to make clear this is specially constructed hash according to the rules dictated by nep461. And I added a comment. I hope this makes it clearer.